src/hera.cpp

```diff
@@ -41,6 +41,12 @@ using namespace std;
 using namespace wasm;
 using namespace HeraVM;
 
+enum hera_wasm_engine {
```

Comment:
1. Why prefix `hera_` instead of `hera` namespace?
2. Prefer `enum class`.

Reply:
1. I guess we can do a cleanup where we move it from `hera_` prefix to `hera` namespace (this applies to already existing code in there so I'd do it after this).

2. What is the difference/benefit of `enum class`?

Reply:
We also use lower case names, leaving upper case only for macros. EVMC is an exception though.